extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -78,14 +79,23 @@ export class AmpGeo extends AMP.BaseElement {
   /** @param {!AmpElement} element */
   constructor(element) {
     super(element);
+    const self = this;
 
     /** @private {number} */
     this.mode_ = mode.GEO_HOT_PATCH;
     /** @private {string} */
     this.country_ = 'unknown';
     /** @private {Array<string>} */
     this.matchedGroups_ = [];
+    /** @private {?function()} */
```

Comment:
Sorry, actual correct type is `?function(!Object<string, (string|Array<string>)>)`

Reply:
The accompanying change to `element-service` should make it work per @jridgewell by waiting until doc ready.  Will fix the type def.